__tests__/app.test.js

```diff
@@ -103,4 +104,47 @@ describe('GET /api', () => {
   });
 
   
+
+  describe('GET /api/articles/:article_id/comments', () => {
+    test('200: responds with an array of comments for the given article_id', () => {  
+      return request(app)
+        .get(`/api/articles/1/comments`)
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toBeInstanceOf(Array);
+          expect(comments.length).toBeGreaterThan(0);
   
+          for (const comment of comments) {
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: 1,
+            });
+          }
+          const timestamps = comments.map(comment => new Date(comment.created_at).getTime());
+        expect(timestamps).toEqual([...timestamps].sort((a, b) => b - a));
```

Comment:
A good check
Might be easier to use a library like `jest-sorted`

Reply:
sure sure, i will update and use jest-sorted